src/channel.c

```diff
@@ -221,9 +223,6 @@ IMPLEMENT_MOCKABLE_FUNCTION(, int, channel_open, THANDLE(CHANNEL), channel)
         /*Codes_SRS_CHANNEL_43_172: [ channel_open shall call srw_lock_acquire_exclusive. ]*/
         srw_lock_acquire_exclusive(channel_ptr->lock);
 
-        /*Codes_SRS_CHANNEL_43_166: [ channel_open shall set is_open to true. ]*/
-        channel_ptr->is_open = true;
-
```

Comment:
Didn't we decide to add this after a whole discussion on how modules that have a list of operations inside them need additional protection?

Reply:
In any case, if we are removing it, the surrounding lock can go too.

Reply:
reverted in next iteration.